onnxruntime/core/session/inference_session.h

```diff
@@ -408,6 +408,9 @@ class InferenceSession {
   ExecutionProviders execution_providers_;
 
  protected:
+  // Threadpool for this session
+  concurrency::ThreadPool* thread_pool_;
```

Comment:
Would've preferred to use a unique_ptr here so that we don't have to remember to call delete in the destructor. In fact, that's how it was setup before.

Reply:
I'll make it be able to passed-in from outside, like OrtEnv. Raw pointer is more flexible in that case.

Reply:
Ok, in that case, please add a comment here saying the same.